docker-compose-rule-core/src/main/java/com/palantir/docker/compose/logging/FileLogCollector.java

```diff
@@ -28,14 +29,15 @@
 import java.util.concurrent.Executors;
 import java.util.concurrent.TimeUnit;
 import org.apache.commons.lang3.Validate;
+import org.joda.time.Duration;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class FileLogCollector implements LogCollector {
 
     private static final Logger log = LoggerFactory.getLogger(FileLogCollector.class);
 
-    private static final long STOP_TIMEOUT_IN_MILLIS = 50;
+    private static final Duration STOP_TIMEOUT = standardSeconds(5);
```

Comment:
Needed to bump this up because 50 milliseconds is not enough time for the log collector to start collecting logs before it is shutdown if we call `DockerComposeRule#after` immediately after calling `DockerComposeRule#before`.